### PR TITLE
Project 4: Building with Backgrounds and Gradients 

### PR DESCRIPTION
**Pull request by Jarfsoft**
## Old Apple Page Clone
- Navigation Bar located with position absolute.
- Added background image to the main section.
- Used grid for the 4 pictures below the main section.
- Footer divided with the copyright and the useful links.

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html]: **Optional**
It's not a bad thing to give your project a befitting title. Please change the title of the project.

File: index.html
Comment (teekaytech) [inline index.html]: Always give your image tags their corresponding `alt` attribute and value. Kindly fix this.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning::vertical_traffic_light: 

### Hello @julian3493 & @Jarfsoft :wave: 

**I must say, you guys have done an awesome job right here :clap:**
- Your linters are green :green_circle: 
- Proper use of CSS grid / flex :100: 
- Nice application of background gradient :+1: 
- Use of semantic tags :ok_hand: 

**Overall, this is looking good. However, there are some issues we need to fix before this project can be approved. Let's get to it:**

**[ REQUIRED CHANGES ]**

- [x] The background image must cover the whole page, no matter the screen size. 
- There is white space around your page. It looks like this is caused by a default margin set by the browser on your `<body>` tag. Kindly find a way to remove this space.





- [x] Each cell in the grid section of the page (4 columns) should be linked. 





- [x] Your page is having a horizontal scroll bar. Kindly identify which of your containers is causing this and remove it.

**Kindly make the required changes and request for another review when you are done.**

*__You can reach out to me on slack (@taofeekolalere) or GitHub (@teekaytech).__*

**Happy coding. :man_technologist:**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! I've see you have done great! :clap: :clap:

## Required Changes

### [Mandatory] :red_circle: 

- [x] Please work on the suggested changes asked for the previous TSE
-  **The background image must cover the whole page, no matter the screen size.**
- **Each cell in the grid section of the page (4 columns) should be linked.**

- [x] Update the link to live preview in the README. This should fix all previous issues.

- [ ] ```<main>``` section should wrap also the 4 bottom images.

### [OPTIONAL]  :large_blue_circle: 

- [ ] Try to use 2 space indentation in HTML and CSS file. This improves readability.

- [ ] Prefer specific CSS selectors over the more general ones.  This improves readability as it assigns meaningful selector names, and this makes it easier to debug. Example:
**Actual**
```
nav a,`
nav input {
   ... some properties ...
}
```
**Improved** :+1: :
```
.nav__link,
.nav__search {
   ... some properties ...
}
```

### [Comments] :green_circle: 

- The README is the most important file of the project, as it describes the project, is important to maintain it updated.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## :clap: :clap: Nice Team!

You've may now merge to master!

### [Comments] :green_circle: 
The page still has a horizontal scroll on smaller screen sizes but because is related to responsiveness is out of the scope of this project goals. 

Still, it is good to mention because this will be an issue for latter projects. Make a habit of using the browsers media queries inspector. This will help you identify if the project looks correct in different screen sizes. Keep in mind that the users may not have the same screen size as the programmer.

An idea to identify the issue of the scrolling horizontal bar: 
image

## See you at the top :rocket:!

**Pull request merged**
